lib/components/dropdown.vue

```diff
@@ -43,15 +43,6 @@
     </div>
 </template>
 
-<style scoped>
-    .dropdown-item:focus,
-    .dropdown-item:hover,
-    .dropdown-header:focus {
-        background-color: #eaeaea;
```

Comment:
could do something similar here by adding `.bv-dropdown` class to root of dropdown, and then targeting like this (with out the need for `scoped`):

```css
.bv-dropdown .dropdown-item:focus:not(.active),
.bv-dropdown .dropdown-item:hover:not(.active),
.bv-dropdown .dropdown-header:focus:not(.active)
.bv-dropdown .dropdown-header:hover:not(.active) {
    /* @See https://github.com/twbs/bootstrap/issues/23329 */
    box-shadow: inset 0px 0px 400px 110px rgba(0, 0, 0, .09);
}

.bv-dropdown .dropdown-item:active,
.bv-dropdown .dropdown-header:active {
    box-shadow: initial;
}
```